Applications/opensim-cmd/test/CMakeLists.txt

```diff
@@ -1,6 +1,6 @@
 
 OpenSimAddTests(
-    TESTPROGRAMS testCommandLineInterface
+    TESTPROGRAMS testCommandLineInterface.cpp
```

Comment:
This change suppresses a CMake warning that complains that the file extension is not explicit.